### PR TITLE
Expose idemix identity cache size in config

### PR DESCRIPTION
**Pull request by yacovm**

### CODE REVIEW

File: token/core/identity/msp/idemix/cache.go [hunk 1]
```diff
@@ -69,16 +111,17 @@ func (c *IdentityCache) Identity(opts *driver2.IdentityOptions) (view.Identity,
 	if logger.IsEnabledFor(zapcore.DebugLevel) {
 		logger.Debugf("fetch identity from backend done [%s][%d]", id, len(audit))
 	}
+
 	return id, audit, nil
 }
 
-func (c *IdentityCache) run() {
+func (c *IdentityCache) provisionIdentities() {
```
[the remark this answers is not in the document]
Comment (yacovm) [reply token/core/identity/msp/idemix/cache.go:118]: this is not the right PR to comment, please comment in the other PR, this PR is about the config

File: token/core/identity/msp/idemix/lm.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ import (
 	"path/filepath"
 	"sync"
 
+	config2 "github.com/hyperledger-labs/fabric-token-sdk/token/core/config"
+
```
Comment (mbrandenburger) [inline token/core/identity/msp/idemix/lm.go:15]: could you please use the chance to get rid of this empty line? Our imports look like minced meat at many places :D

**Review by mbrandenburger — CHANGES_REQUESTED** [review]
LGTM! This PR seems to require #306 to be merged first.

File: token/core/identity/msp/idemix/lm.go (hunk 2 above)
Comment (yacovm) [reply token/core/identity/msp/idemix/lm.go]: done

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**